src/map/script.c

```diff
@@ -21540,6 +21572,8 @@ BUILDIN(bindatcmd)
 		atcommand->binding[i]->group_lv = group_lv;
 		atcommand->binding[i]->group_lv_char = group_lv_char;
 		atcommand->binding[i]->log = log;
+		CREATE(atcommand->binding[i]->at_groups, char, db_size(pcg->db));
```

Comment:
What do you suggest then? looping through every member of the db? I thought it was correct because already checked elsewhere: https://github.com/mekolat/Hercules/blob/master/src/map/pc_groups.c#L146